### ISSUE TITLE
Strike Out Breaks On URLs

### ISSUE THREAD

**Issue by aberfan** [pbek/QOwnNotes#2755]
~~strikeing out~~ notes that have a url won't work if it's a proper url.
```
~~test  [http://test.com/]~~

~~[http://this is a test.com]~~
```
![e5b375a8ae55d48736e1bededf46624d](https://user-images.githubusercontent.com/72705523/226085050-2d6536e7-b14c-4827-8671-80afa7ede5fa.png)

Adding a backslash before the period in the first one _will_ allow it to be properly struck out.

**Comment by aberfan** [comment]
I have many lines of information, all which also have a URL.

I want to visually note some of those lines aren't relevant, but I need to keep the information in them.  The cruddy way of doing it is to do a sub header and move the lines (which are in chronological order, and is important that they are) to visually note they're not used.

Or I can strike-out.  The strike-out works well for sets (these are monthly) without URLs.  However, any line with a proper URL breaks as demonstrated in the screenshot.

Expected behavior is that if I type:
\~\~test  [http://test.com/]\~\~
that it will be struck-out.

**Comment by pbek** [comment]
Seems like an issue of https://github.com/mity/md4c...

**Comment by pbek** [comment]
> Seems like an issue of https://github.com/mity/md4c...

Nah, the html is correct: `<del><a href="http://test.com/">http://test.com/</a></del>`. So it's an issue of the [QTextBrowser](https://doc.qt.io/qt-5/qtextbrowser.html) widget. Then we are out of luck at the moment...

**Comment by pbek** [comment]
Working, as of 26.2.13:

<img width="475" height="180" alt="Image" src="https://github.com/user-attachments/assets/edecf1cd-118e-4648-beb9-a0d789ea94db" />
